test/suites/drift/suite_test.go

```diff
@@ -272,12 +274,12 @@ var _ = Describe("Drift", Label("AWS"), func() {
 				return nodes[i].CreationTimestamp.Before(&nodes[j].CreationTimestamp)
 			})
 			nodeTwo := nodes[1]
+			// Remove the startup taints from the new nodes to initialize them
 			Eventually(func(g Gomega) {
-				n := &v1.Node{}
-				g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(nodeTwo), n)).To(Succeed())
-				stored := n.DeepCopy()
-				n.Spec.Taints = []v1.Taint{}
-				g.Expect(env.Client.Patch(env.Context, n, client.MergeFrom(stored))).To(Succeed())
+				g.Expect(env.Client.Get(env.Context, client.ObjectKeyFromObject(nodeTwo), nodeTwo)).To(Succeed())
+				stored := nodeTwo.DeepCopy()
+				nodeTwo.Spec.Taints = lo.Reject(nodeTwo.Spec.Taints, func(t v1.Taint, _ int) bool { return t.Key == "example.com/another-taint-2" })
```

Comment:
QQ: Was this racing with the Unschedulable taint and cleanup?

Reply:
Discussed offline. This was fixed because the old code was just wrong. This wasn't catching a flake but the old code _might_ have had the chance to flake.